include/openenclave/bits/evidence.h

```diff
@@ -78,6 +78,11 @@ extern const char* OE_REQUIRED_CLAIMS[OE_REQUIRED_CLAIMS_COUNT];
 // This array is needed for tests
 extern const char* OE_OPTIONAL_CLAIMS[OE_OPTIONAL_CLAIMS_COUNT];
 
+/**
+ * Additional claims related to TLS certificate
+ */
+#define OE_CERTIFICATE_PUBLIC_KEY "public_key"
```

Comment:
If this is a claim name then say so in the #define name.  Other claim names all start with OE_CLAIM_ 
Also the others are in lines 61-77, not separated by what they're "related to".   Put here, it implies it's neither required nor optional.  Not sure what that would mean or why it's in neither array.

Reply:
This is what OE attestation plugin framework calls `custom claims`. In fact, it's a stationary claim in attested TLS in the way that:

1) the attester generates a public/private key pair
2) the attester generates a report, using the hash of the public key as the report data
3) the attester generates a certificate with the report embedded.

On the relying party side:
1) the verifier validates the certificate is signed properly
2) the verifier extracts the report and validates it is signed by a TEE (SGX in this case)
3) the verifier validates the hash of the public key in the certificate matches the report data

With all 3 steps passed, the public key is used for subsequent TLS communications.

In short, this claim neither required nor optional for remote attestation purpose. But it is required for attested TLS or any certificate based attestation.